src/core/public/application/types.ts

```diff
@@ -243,6 +243,24 @@ export interface LegacyApp extends AppBase {
   disableSubUrlTracking?: boolean;
 }
 
+/**
+ * Information about a registered {@link App | application}
+ *
+ * @public
+ */
+export type AppInfo = Omit<App, 'mount' | 'updater$'> & {
+  legacy: false;
+};
```

Comment:
Was not very inspired when choosing a name for these types. Any better suggestion is welcome

Reply:
`PublicAppInfo` to indicate that's publicly available to other plugins?